staging/src/k8s.io/legacy-cloud-providers/OWNERS

```diff
@@ -1,16 +1,18 @@
 # See the OWNERS docs at https://go.k8s.io/owners
-
+# We are no longer accepting features into k8s.io/legacy-cloud-providers.
+# Any kind/feature PRs must be approved by SIG Cloud Provider going forward.
+options:
+  no_parent_owners: true
 approvers:
-- mikedanese
-- dims
 - andrewsykim
 - cheftako
-reviewers:
-- mikedanese
 - dims
+- liggitt
```

Comment:
maybe add a comment above or beside these names indicating they're here for code organization / dependency update approvals

Reply:
done